packages/winpanda/extra/src/winpanda/core/constants.py

```diff
@@ -22,3 +22,13 @@
 
 # List of package installation config file extensions
 PKG_INSTCFG_FEXTS = [PKG_EXTCFG_FEXT, PKG_SVCCFG_FEXT]
+
+# Package's extra installation helper files
+PKG_EINSTHLP_STDOUT_FEXT = 'stdout'
+PKG_EINSTHLP_STDERR_FEXT = 'stderr'
+PKG_EINSTHLP_STDOUT_FNAME = '.'.join([
+    'pkg-einst-helper-{id}', PKG_EINSTHLP_STDOUT_FEXT
+])
+PKG_EINSTHLP_STDERR_FNAME = '.'.join([
+    'pkg-einst-helper-{id}', PKG_EINSTHLP_STDERR_FEXT
```

Comment:
What is an "einst"? Can we just call it `pkg-install-1.stderr`?

Reply:
**'pkg-einst-helper'** stands for **'Package extra installation helper'**, which is exactly corresponds functional designation of that component. **'pkg-install'** name won't reflect real meaning/source of that artifact. 
Would it help, if we change it to **'pkg-extra-installation-helper'**?